models/issues/issue_list_test.go

```diff
@@ -67,9 +67,7 @@ func TestIssueList_LoadAttributes(t *testing.T) {
 		if issue.ID == int64(1) {
 			assert.Equal(t, int64(400), issue.TotalTrackedTime)
 			assert.NotNil(t, issue.Project)
-		} else if issue.ID == int64(2) {
-			assert.Equal(t, int64(3682), issue.TotalTrackedTime)
-			assert.Nil(t, issue.Project)
```

Comment:
Do we needs these checks

Reply:
We usually don't alter backports. If it's an issue, it should have been fixed in the original PR, or a followup to it.

Reply:
At first, we have 3 issues in the list, but issue(ID:2) was removed in: 
https://github.com/go-gitea/gitea/pull/10814/files#diff-8f8d49e4029a3d68849f2af16dd68a6dcbd2bb25d9ba6d6b906185c1a8cb0f3cL37
But in that PR, he only removed the issue in the issue list, and forgot removing the test code below.

ps: I have tried to add it back, but ci failed. So maybe we don't need to test issue(ID:2) here any more.